.eslintrc.js

```diff
@@ -24,6 +24,12 @@ module.exports = {
       },
       rules: {
         'node/no-unpublished-require': 'off',
+        'node/no-missing-require': [
+          'error',
+          {
+            allowModules: ['@strapi/strapi'],
```

Comment:
this should be fine since `@strapi/strapi` is a peerDependency

Reply:
Looks good